test/fixtures/monthlyOffer.json

```diff
@@ -1,4 +1,5 @@
 {
+  "id": 1,
```

Comment:
It's always a good habit to use random-ish numbers for things like test ID's so it's less likely to get passing values just by accident. Especially make sure all the offers have different ID's so if the wrong property is used somewhere, the tests can catch it.